internal/service/cloudwatch/composite_alarm.go

```diff
@@ -43,6 +43,28 @@ func ResourceCompositeAlarm() *schema.Resource {
 				Default:  true,
 				ForceNew: true,
 			},
+			"actions_suppressor": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"alarm": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: validation.StringLenBetween(1, 1600),
```

Comment:
I'm always careful with these ValidateFunc's, even though the API says 1 - 1600 it's often added code we have to maintain.

In this case I don't think it's a worthy validation step, as this can change at any given moment. Leaving it out I think is fine, as the API would throw the error anyways.

Reply:
I agree, good point I removed this validation.